tools/gen_esp32part.py

```diff
@@ -246,14 +278,14 @@ def find_by_name(self, name):
                 return p
         return None
 
-    def verify(self):  # noqa: C901
+    def verify(self):
         # verify each partition individually
         for p in self:
             p.verify()
 
         # check on duplicate name
         names = [p.name for p in self]
-        duplicates = set(n for n in names if names.count(n) > 1)  # noqa: C401
+        duplicates = set(n for n in names if names.count(n) > 1)
```

Comment:
change to `duplicates = {n for n in names if names.count(n) > 1}`

Reply:
Ah yes, forgot the branch is in the organization. Will apply the fix tomorrow. Thanks!